projects/cincinnati/cincinnati.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: List
```

Comment:
nit: no need for a list here

Reply:
the list is needed but I didn't have the `ImageStream` in the version you commented on